zipkin-storage/mysql/README.md

```diff
@@ -41,6 +41,38 @@ For example, all the below query the same trace using different tools:
 * zipkin-api: `http://1.2.3.4:9411/api/v1/trace/27960dafb1ea7454?raw`
 * mysql: `select * from zipkin_spans where trace_id = x'27960dafb1ea7454';`
 
+If you are trying to debug from data in the database, it is helpful to
+format IDs as hex, and timestamps as dates. The following is an example
+query which will return one line for each update to a span in the last
+5 minutes.
+
+```sql
+SELECT lower(concat(CASE trace_id_high
```

Comment:
while most ppl use 64bit trace ids, this part of the query will match what's in the json or B3 headers